config/settings/base.py

```diff
@@ -113,7 +113,7 @@
 
 LANGUAGE_CODE = 'en-us'
 
-TIME_ZONE = 'UTC'
+TIME_ZONE = 'America/Phoenix'
```

Comment:
Ah, I misread your comment! Yeah, let me stuff in timezone for the workshop dates real quick.